### PR TITLE
feat(onboard/postcss vuejs): postcss config for vue.js

### PR DESCRIPTION
**Pull request by tiloyi**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/contributing)

- [x] Yes
- [ ] No

## Does this PR introduce a breaking change?

read [What is a breaking change ?](/Contributing/GitConventions/)

- [ ] Yes
- [x] No

## Describe the changes

Add PostCSS config for Vue.js on "Getting started"

Github issue number or Jira issue URL: **MZC-168**

### CODE REVIEW

File: src/docs/GetStarted/Developers/configurePostCss/index.mdx [hunk 1]
```diff
@@ -70,3 +70,29 @@ gulp.task('css', () =>
...
+You will need to use **PostCSS** via `postcss-loader`.
+
+```npm
+npm install -D postcss-loader
+```
+
+After installing `postcss-loader`, configure the file `vue.config.js` this way:
+
+```js
+const postcssPluginConfig  = require('@mozaic-ds/css-dev-tools/postcssPluginConfig')
```
Comment (gael-boyenval) [inline src/docs/GetStarted/Developers/configurePostCss/index.mdx]: one line break

File: src/docs/GetStarted/Developers/configurePostCss/index.mdx [hunk 2]
```diff
@@ -70,3 +70,29 @@ gulp.task('css', () =>
...
+module.exports = {
+  lintOnSave: false,
```
Comment (gael-boyenval) [inline src/docs/GetStarted/Developers/configurePostCss/index.mdx]: remove this line

File: src/docs/GetStarted/Developers/configurePostCss/index.mdx [hunk 3]
```diff
@@ -70,3 +70,29 @@ gulp.task('css', () =>
     .pipe(gulp.dest('./dest'))
 )
 ```
+
+### C) with Vue.js
```
Comment (gael-boyenval) [inline src/docs/GetStarted/Developers/configurePostCss/index.mdx]: With Vue.js using vue CLI

File: src/docs/GetStarted/Developers/configurePostCss/index.mdx (hunk 1 above)
Comment (tiloyi) [reply src/docs/GetStarted/Developers/configurePostCss/index.mdx]: fixed

File: src/docs/GetStarted/Developers/configurePostCss/index.mdx (hunk 2 above)
Comment (tiloyi) [reply src/docs/GetStarted/Developers/configurePostCss/index.mdx]: fixed

File: src/docs/GetStarted/Developers/configurePostCss/index.mdx (hunk 3 above)
Comment (tiloyi) [reply src/docs/GetStarted/Developers/configurePostCss/index.mdx]: fixed

**Pull request merged**